pkg/storaged/content-views.jsx

```diff
@@ -327,7 +346,63 @@
         });
     }
 
+    function repair() {
+        const vgroup = lvol && client.vgroups[lvol.VolumeGroup];
+        if (!vgroup)
+            return;
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20230918-111530-a92123e3-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)